### DISCUSSION TITLE
logging events sent to room onAnyOutgoing

### DISCUSSION THREAD

**Discussion by aravindsakthivel** [socketio/socket.io#4535] [Q&A] [answered]
When I'm trying to log the event details sent to the room through `onAnyOutgoing`, the event was printing multiple times which was equal to no of users in the room. is it possible to log the event sent to the room only once?

**Comment by darrachequesne** [comment, marked as answer]
Hi! Unfortunately, this is not currently possible. Though if there is enough interest we might add this feature in the future.
